app/controllers/items/new.js

```diff
@@ -69,6 +70,23 @@ export default GoodcityController.extend(
       return this.get("i18n").t(str);
     },
 
+    allAvailablePrinter: Ember.computed(function() {
+      return this.get("printerService").allAvailablePrinter();
```

Comment:
@akshaygwd `allAvailablePrinter` should be `allAvailablePrinters` because its gonna return an array of printers.

Reply:
Same comment for this computed property.